selfdrive/modeld/navmodeld.cc

```diff
@@ -41,6 +42,11 @@ void run_model(NavModelState &model, VisionIpcClient &vipc_client) {
 int main(int argc, char **argv) {
   setpriority(PRIO_PROCESS, 0, -15);
 
+  // there exists a race condition when two processes try to create a
+  // SNPE model runner at the same time, wait for dmonitoringmodeld to finish
+  LOGW("waiting for dmonitoringmodeld to initialize");
+  Params().getBool("DmModelInitialized", true);
```

Comment:
getBool(param, block) will return false if SIGINT caught. 
it might be better to exit main if it return false.